audb/core/load.py

```diff
@@ -887,17 +876,16 @@ def load(
     if not removed_media:
         _remove_media(db, deps, num_workers, verbose)
 
+    # Adjust full paths and file extensions in tables
+    if full_path:
+        root = db_root
+    else:
+        root = None
+    _update_path(db, db_root, full_path, flavor.format, num_workers, verbose)
```

Comment:
Do you want to pass `root` here?

Reply:
No, sorry, code was not ready. Removed now the `if`-`else` statement.